app/services/Config.scala

```diff
@@ -163,6 +164,9 @@ class DevConfig extends Config {
     "https://targeting.local.dev-gutools.co.uk",
     "https://campaign-central.local.dev-gutools.co.uk"
   )
+  override def corsablePostDomains: Seq[String] = Seq(
+    "https://targeting.local.dev-gutools.co.uk"
```

Comment:
We can pull these URLs out into a `val` and reference them in the function so that they are not created every time the function is invoked.